docs/examples/unit_tests/execute.sh

```diff
@@ -24,208 +24,224 @@
 #
 #	Variable definitions
 #	--------------------
-#	testLog		temporary test log file that holds compile and run-time logs from each test
-#	log		temporary log file that collects compile and run-time logs from all tests
-#	marks		array containing marks from all the tests
-#	comments	"Wrong Answer", "Compilation Error", "Timeout", "Accepted"
-#	comments to be added in future: "Runtime Error", "Partial Answer", "Exception", "Files Not Available",
-#					"Unsupported Language"
 #
-#	testDir		directory containing all the relevant tests for each language supported for an assignment;
+# All variables that are exported/imported are in upper case convention. They are:
```

Comment:
Please make sure that this file is same as the execute.sh file found in docs/examples/io_tests directory.